Products/ZenHub/metricpublisher/tests/test_publisher.py

```diff
@@ -0,0 +1,441 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2014, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+from unittest import TestCase, TestSuite, makeSuite
+from mock import Mock, MagicMock, create_autospec, patch
+from zope.interface.verify import verifyObject
+
+import ujson as json
```

Comment:
The module under test imports already imports json, so it should be imported from Products.ZenHub.metricpublisher.publisher